pkg/migration/migrations/separated_intents.go

```diff
@@ -407,7 +482,7 @@ func runSeparatedIntentsMigration(
 }
 
 func separatedIntentsMigration(
-	ctx context.Context, cv clusterversion.ClusterVersion, deps migration.SystemDeps, _ *jobs.Job,
+	ctx context.Context, cv clusterversion.ClusterVersion, deps migration.SystemDeps, job *jobs.Job,
```

Comment:
Can we add `job` to `migration.SystemDeps`?

Reply:
It didn't feel like it belonged there to me: mentioned on #69670 that the `SystemDeps` is a static bag of dependency APIs/services that is passed to any migration, while the job is an argument to the individual migration, just like `cv`